app/controllers/place_controller.rb

```diff
@@ -27,11 +28,16 @@ def publication
   end
 
   def locals
-    if params[:slug] == REPORT_CHILD_ABUSE_SLUG
+    case params[:slug]
+    when REPORT_CHILD_ABUSE_SLUG
       {
         option_partial: "option_report_child_abuse",
         preposition: "for",
       }
+    when FGM_SLUG
```

Comment:
This seems like a neat way to get people straight to the results section. Is there a reason that this change is being limited to one route? Could `results_anchor: "results"` not be appended to locals in all cases?

Reply:
It could be appended in all cases, it's just that [this](https://www.gov.uk/female-genital-mutilation-help-advice) was the specific page highlighted where this was an issue and I wasn't sure if I should roll it out to all pages.